kwasm

```diff
@@ -59,22 +63,25 @@ run_kast() {
 }
 
 run_prove() {
-    export K_OPTS=-Xmx8G
-    haskell_backend_command="$k_release_dir/bin/kore-exec"
-    if $repl; then
-        haskell_backend_command="$k_release_dir/bin/kore-repl --repl-script $kwasm_dir/kast.kscript"
-    fi
-    kprove --directory "$backend_dir" "$run_file" "$@" --haskell-backend-command "$haskell_backend_command"
+    local additional_proof_args
+
+    def_module="$1" ; shift
+
+    additional_proof_args=()
+    ! $repl       || additional_proof_args+=(--debugger --debug-script kast.kscript)
```

Comment:
Can you remove the `--debug-script`, that file isn't available anymore.

Reply:
Done.